examples/with-typescript-graphql/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": false,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
Can we use `strict` in this example? it's very useful

Reply:
Absolutely. I think I'll use the same config of [with-typescript](https://github.com/zeit/next.js/blob/canary/examples/with-typescript/tsconfig.json).